### PR TITLE
Assignment 1 Completed

### PR DESCRIPTION
**Pull request by tianxie812**

### CODE REVIEW

**Comment by ldconejo** [comment]
Hi Tian,

For lesson 1, you only included _main.py_ and your tests in your submission. Could you make sure you commit all of your code for review? 

Thanks,

Luis

File: students/Tianx/Lesson2/charges_calc.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    FILE_HANDLER.setFormatter(FORMATTER)
+
+    CONSOLE_HANDLER = logging.StreamHandler()
+    CONSOLE_HANDLER.setFormatter(FORMATTER)
+    LOGGER = logging.getLogger()
+    LOGGER.addHandler(FILE_HANDLER)
+    LOGGER.addHandler(CONSOLE_HANDLER)
+
+    if debug_level == '0':
+        LOGGER.disabled = True
```
Comment (ldconejo) [inline students/Tianx/Lesson2/charges_calc.py:36]: Note that with -d 0 your code is still showing logging messages. Try to fix that. Consider the possibility of assigning CRITICAL to level 0:

```
    LOGGER.setLevel(logging.CRITICAL)
    FILE_HANDLER.setLevel(logging.CRITICAL)
    CONSOLE_HANDLER.setLevel(logging.CRITICAL)
```

File: students/Tianx/Lesson2/charges_calc.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        LOGGER.setLevel(logging.WARNING)
+        FILE_HANDLER.setLevel(logging.WARNING)
+        CONSOLE_HANDLER.setLevel(logging.WARNING)
+    elif debug_level == '3':
+        LOGGER.setLevel(logging.DEBUG)
+        FILE_HANDLER.setLevel(logging.WARNING)
+        CONSOLE_HANDLER.setLevel(logging.DEBUG)
+    else:
+        logging.debug("ValueError!")
+        logging.error("Debug level should be 0-3")
```
Comment (ldconejo) [inline students/Tianx/Lesson2/charges_calc.py:51]: Your program should stop execution if the user tries an invalid logging level, so add something like:

`sys.exit()`

**Comment by ldconejo** [comment]
Hi Tian,

See my comments on your code for lesson 2. Also, remember to run Pylint on your code and address any issues reported. Please submit a new pull request once you have made the required changes.

Regards,

Luis

**Comment by tianxie812** [comment]
Hi Luis,

I thought I submitted all my lesson1 scripts with the pull request #749 and it was merged. I am creating this pull request at the root directory level, will that synchronize my entire folder: "Tianx"?
I believe when I submitted the assignment on the website, I included a zip file with everything in it.

Thanks,
Tian

**Pull request closed without merging**